source/validate_instruction.cpp

```diff
@@ -119,7 +119,6 @@ spv_result_t CapCheck(ValidationState_t& _,
 // clang-format off
 spv_result_t InstructionPass(ValidationState_t& _,
                              const spv_parsed_instruction_t* inst) {
-  if (_.is_enabled(SPV_VALIDATE_INSTRUCTION_BIT)) {
     if (inst->opcode == SpvOpCapability)
```

Comment:
I would expect this to be reformatted to the left since an indentation level has disappeared.